src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -511,6 +511,13 @@ def list_workflow_runs(
 
     # Resolve config
     resolved_config: RuntimeConfig = _resolve_config(config)
+    # If user wasn't specific with workspace and project, we might want to resolve it
+    if not workspace and not project:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** This is probably a style thing, I prefer to explicitly check for `None`.‏
```suggestion
    if workspace is not None and project is not None:
```